src/hostapi/oboe/pa_oboe.cpp

```diff
@@ -363,506 +380,391 @@ bool OboeEngine::tryStream(Direction direction, int32_t sampleRate, int32_t chan
  * @return  paNoError if everything goes as expected, paUnanticipatedHostError if Oboe fails to open
  *          a stream, and paInsufficientMemory if the memory allocation of the buffers fails.
  */
-PaError OboeEngine::openStream(Direction direction, int32_t sampleRate,
-                               Usage androidOutputUsage, InputPreset androidInputPreset) {
-    PaError m_error = paNoError;
-    Result m_result;
-
-    if (direction == Direction::Input) {
-        inputBuilder.setChannelCount(oboeStream->bufferProcessor.inputChannelCount)
-                ->setFormat(PaToOboeFormat(oboeStream->inputFormat))
-                ->setSampleRate(sampleRate)
+PaError OboeEngine::openStream(OboeStream *i_oboeStream, Direction i_direction, int32_t i_sampleRate,
```

Comment:
This "i_" prefix does not conform to PortAudio or Android style guidelines.
Please see:  https://github.com/PortAudio/portaudio/wiki/ImplementationStyleGuidelines#naming-conventions